src/components/Tiles/MovieTile.js

```diff
@@ -1,12 +1,20 @@
 import { Star } from '../Star/Star';
 import { ImageComponent } from '../ImageComponent/ImageComponent';
 import ApiProvider from '../../providers/ApiProvider';
+import { observerListener } from '../../shared/observer';
+import { routes } from '../../route/routes';
+import { redirect } from '../..';
 
 export const MovieTile = async ({ movieID }) => {
     const movie = await ApiProvider.getTitleDetails(movieID);
 
+    observerListener(`movie-tile-${movie.id}`, () => {
+        const TileOnclick = routes.details.getPathWithParams(movie.id);
```

Comment:
1. can't you just pass this function to the ImageCompoment? Wouldn't it work?
2. Remove unused imports
3. "TileOnclick", name is not accurate, it's not a function, just a constant, we are using camel case so starting with capital letter is wrong

Reply:
But yeah, if you can't go with the onClick on ImageComponent, leave it as it is, just fix this 2 and 3 point and I'm gonna approve 8)

Reply:
2. it seems all imports are used.
3. name has been changed, I belive now it will be better